force-app/main/default/chat/lwc/chat/chat.js

```diff
@@ -0,0 +1,85 @@
+//todo:: const labels in html
+import {LightningElement, track, wire} from 'lwc';
+import {createRecord, getRecord} from "lightning/uiRecordApi";
+import {subscribe, unsubscribe} from 'lightning/empApi';
+
+import notifyAboutMessageApex from '@salesforce/apex/ChatController.notifyAboutMessage';
+
+import MESSAGE_OBJECT from "@salesforce/schema/Message__c";
+import BODY_FIELD from "@salesforce/schema/Message__c.Body__c";
+
+import MESSAGE_ID_FIELD from "@salesforce/schema/Chat_Event__e.Message_Id__c";
+
+import {MessageWrapperBuilder} from "c/messageItem";
+
+export default class Chat extends LightningElement {
+    /**
+     * @type {MessageWrapper[]}
+     */
+    @track messages = [];
+    receivedMessageId = '';
+    messageFields = ['Message__c.Id', 'Message__c.Body__c', 'Message__c.User_Alias__c', 'Message__c.CreatedDate'];
+
+    channelName = '/event/Chat_Event__e';
+    subscription;
+
+    connectedCallback() {
+        this.subscribeOnChatEvent()
+    }
+
+    async handleSend(event) {
+        event.stopPropagation();
+        const fields = {};
+
+        fields[BODY_FIELD.fieldApiName] = event.detail;
+
+        const recordInput = {
+            apiName: MESSAGE_OBJECT.objectApiName,
+            fields: fields
+        };
+
+        let messageId;
+        await createRecord(recordInput).then((record) => {
+            messageId = record.id;
+        });
+        await this.notifyAboutMessageApex(messageId);
+    }
+
+    async notifyAboutMessageApex(messageId) {
+        await notifyAboutMessageApex({messageId: messageId});
+    }
+
+    subscribeOnChatEvent() {
+        const thisRef = this;
```

Comment:
Do you like it this way? You could eliminate this variable using an arrow function or the bind.

Reply:
thanks! changed 02061e6dbde3026302eaa29468fca3f8c5eefa79